stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/FormElement.kt

```diff
@@ -21,6 +22,13 @@ interface FormElement {
      */
     val allowsUserInteraction: Boolean
 
+    /**
+     * The mandate text, if any.
+     *
+     * This is used to determine if vertical mode needs to show mandates for LPMs with no fields, only a mandate.
+     */
+    val mandateText: ResolvableString?
```

Comment:
Did you consider including the `mandateText` in `SupportedPaymentMethod`? It makes sense to me to include it there in the same way we include `subtitle` and `mandateText` feels a little out of place in the FormElement

Reply:
I _mostly_ did it to leave a single source to truth, similar to why I added `allowsUserInteraction` to `FormElement`.

`subtitle` is a net new thing, and uses different localizations than the "subtitle".

I'm not hard set on this. The implementation for adding it to `SupportedPaymentMethod` would be quite a bit simpler!

Reply:
Talked more offline.

I think we need to do this approach because we get mandates back from LUXE, which would add a lot more complexity to adding to to SupportedPaymentMethod (it would basically end in the same implementation).